src/test/java/org/ays/auth/tests/PasswordForgotTest.java

```diff
@@ -34,4 +36,26 @@ public void forgotPasswordNegativeTest(String emailAddress, AysErrorMessage erro
                 .spec(AysResponseSpecs.subErrorsSpec(errorMessage, field, type));
     }
 
+    @Test(groups = {"Smoke", "Regression"})
+    public void forgotPasswordVerifyTimeChangeInResponse() {
+        PasswordForgotPayload passwordForgotPayload = new PasswordForgotPayload();
+        String emailAddress = UserDataSource.findAnyEmailAddress();
+        passwordForgotPayload.setEmailAddress(emailAddress);
+
+        Response firstResponse = AuthEndpoints.forgotPassword(passwordForgotPayload);
+        firstResponse.then()
+                .spec(AysResponseSpecs.expectSuccessResponseSpec());
+
+        Response secondResponse = AuthEndpoints.forgotPassword(passwordForgotPayload);
+        secondResponse.then()
+                .spec(AysResponseSpecs.expectSuccessResponseSpec());
+
+        String firstResponseTime = firstResponse.jsonPath().getString("time");
+        String secondResponseTime = secondResponse.jsonPath().getString("time");
+
+        assertNotEquals(firstResponseTime, secondResponseTime,
+                "The time value is the same in the first and second response.");
+
+    }
+
```

Comment:
Selam @cagla88 , ben bu senin yapmış oldugun sekilde ayrı bir test ile zamanı test etmenin daha yerinde oldugunu düşünüyorum. Aksi durumda her request oncesi bir time objesi oluşturup bunu reponse'daki time ile karsilastirma gibi bir yola gitmek gerekir. @agitrubard 'in de bu konudaki fikrini merak ediyorum. 
Diger değinmek istediğim şeyler:
1. Acaba time formatını da test etmemize gerek var mi asagida belirttiğim gibi bir method ile:
```
private boolean isValidISO8601Time(String time) {
    String iso8601Regex = "\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d{1,9}";
    return time != null && time.matches(iso8601Regex);
}
```
2.Rastgele bir user seçip şifre sıfırlamak yerine sıfırdan bir user oluşturup şifre sıfırlama islemi yapilsa daha iyi olabilir, aksi taktirde test userlarimiz calismadiginda neden kaynaklandiigini tespit etmek ile vakit kaybetmiş oluruz.

Reply:
@Nbhtozcn selamlar, 2. maddede belirttiğiniz şekilde güncellerim. 1. madde ile ilgili de olsun diyorsanız onuda ekleyebilirim. Böyle method olarak değilde AysResponseSpecs class içerisine String object tanımlayıp `code` alanında olduğu şekilde spec olarak eklemek daha iyi olabilir

   ```
    private static final String TIME_FORMAT_REGEX = "\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d{1,9}";
    public static ResponseSpecification expectSuccessResponseSpec() {
        return new ResponseSpecBuilder()
                .expectStatusCode(200)
                .expectContentType("application/json")
                .expectBody("time", notNullValue())
                .expectBody("time", matchesPattern(TIME_FORMAT_REGEX))
                .expectBody("code", matchesPattern(UUID_REGEX))
                .expectBody("isSuccess", equalTo(true))
                .build();
    }
```

bu şekilde uygun mu sizin içinde?

Reply:
Uygun oldugunu düşünüyorum @cagla88 Eline sağlık